overlay/files-nvidia/system/oem/mount.yaml

```diff
@@ -0,0 +1,11 @@
+## TODO: this is a workaround
+## The orin packages are writing to /usr/local, which is mounted to COS_PERSISTENT.
+## We probably should run this in immucore, overlaying the /usr/local of the image to COS_PERSISTENT.
+## For the time being, doing it in the cloud configs.
+#
+### Note: This have the consequences of everything stored inside /usr/local to go inside the "local" directory inside the partition.
+###       Usually this is not the case as there is no need of subtrees, but due to how overlayfs work the workdir and uppermount needs to be in the same filesystem.
+stages:
+  initramfs.before:
+  - commands:
+    - umount /usr/local && mkdir -p /run/mount/persistent && mount /dev/disk/by-label/COS_PERSISTENT /run/mount/persistent && mkdir -p /run/mount/persistent/work /run/mount/persistent/local && mount -t overlay -o lowerdir=/usr/local,upperdir=/run/mount/persistent/local,workdir=/run/mount/persistent/work /usr/local
```

Comment:
is there a way to do this in multiple lines, just for readability? not a big deal though

Reply:
good point, done in https://github.com/kairos-io/kairos/pull/1562/commits/77e767e7666c7046edba0a25722e432a11140583